### ISSUE TITLE
Show reminder (with EDIT) above category

### ISSUE THREAD

**Issue by licaon-kter** [SecUSo/privacy-friendly-notes#25]
Now you need to touch icon->Edit just to see it.

**Comment by Yonjuni** [comment]
Could you describe this in a bit more detail? 
In which view should it be placed? 
What should the reminder remind?

**Comment by licaon-kter** [comment]
How do you call that thing that you set up with the clock icon? I call it a "reminder".

I want to see that when I open a note, not hidden away behind 5 more actions (clock>edit>day>hour>cancel)

**Comment by Yonjuni** [comment]
Okay, so the way to the reminder is too long and it could be unclear too a user.

**Comment by licaon-kter** [comment]
Yes, you open a note and kind of expect to see it's content and properties there.

**Comment by ecanja** [comment]
Just discovered this app and like it a lot - it's very clearly arranged.
The only thing I noticed as well is that you cannot see the date and time of a reminder once you've set it up. It would be great to have this, so hereby I'm backing licaon-kter's feature request. :-)
